packages/telemetry/src/events/session.ts

```diff
@@ -36,6 +36,8 @@ interface EventCliSessionInternal extends EventCliSession {
 	config?: ConfigInfo;
 	configKeys?: string[];
 	flags?: string[];
+	optionalIntegrations?: number;
+	totalNumOfIntegrations?: number;
```

Comment:
<insert bike shed joke>

nit: Might be nice to have the names look similar, maybe just `totalIntegrations` or `optionalIntegrationsCount` and `integrationsCount`?